src/components/HomeLicenseFilter.vue

```diff
@@ -40,7 +40,7 @@ export default {
 
 <style lang="scss" scoped>
 .home-license-filter {
-  margin-top: 0.5em;
+  margin-top: 1.5em;
```

Comment:
You should use the spacing margin classes in Vocabulary https://cc-vocabulary.netlify.com/?path=/docs/tokens-spacing--figma.

This margin doesn't add to 64px and it should be on desktop only. Mobile margin should be lower because it can cut off on smaller screens